ngonoessono_a05a.js

```diff
@@ -0,0 +1,44 @@
+/*Gertrude Vanessa Ngono Essono
+ngonoessono_a05a.js
+INFO2124
+Thoendel
+01/172020*/
+
```

Comment:
I would have liked to see comments line by line throughout the program explaining what each line is doing. As per the assignment - the absence of these comments is worth 10%:
-2